src/IssueViz.Security/OpenInIDE/Http/HttpListenerFactory.cs

```diff
@@ -74,19 +80,22 @@ IDisposable IListenerFactory.Create(int startPort, int endPort)
             return listener;
         }
 
-        private static IDisposable CreateListener(int port)
+        private IDisposable CreateListener(int port)
         {
             var appProperties = new AppProperties(new Dictionary<string, object>())
             {
                 Addresses = AddressCollection.Create()
             };
 
             var address = Address.Create();
+            address.Scheme = "http";
+            address.Host = "localhost";
             address.Port = port.ToString();
+            address.Path = "/sonarlint/api/";
```

Comment:
I've tried to make the API as close to the style used by SQ. I'll document it in the spec document so we can discuss it with the rest of team and the SQ folk.